packages/admin/src/components/Navigation/StyledComponents.js

```diff
@@ -1,53 +1,52 @@
 import styled from 'styled-components'
 import {Link} from 'react-router-dom'
-
-export const StyledSearch = styled.input`
-  border-radius: 20px;
-  border: 0px;
-  width: 90%;
-  font-size: 1.2rem;
-  padding: 4px 7px;
-  :focus {
-      outline: none;
-  }
-  margin-bottom: 5px;
-`
+import {Button, theme} from 'tocco-ui'
 
 export const StyledNav = styled.nav`
   height: 100%;
 `
 
-export const StyledMewnuWrapper = styled.div`
+export const StyledMenuWrapper = styled.div`
   height: 100%;
   overflow: auto;
+  padding-left: 2rem;
+  
+  div > div:last-child {
+    padding-bottom: 2rem;
+  }
+
   ::-webkit-scrollbar {
-      width: 5px;
+      width: 10px;
```

Comment:
i think we should move a scrollbar css into tocco-ui since we want to reuse the same scrollbar in various views.